Traditional_Morison_Element/README.md

```diff
@@ -0,0 +1,11 @@
+# Morison Element
+
+**Author:**  	Adam Keester
+
+**Version:** 	WEC-Sim v4.3
+
+**Geometry**	Monopile
+
+Example on using Traditional Morison Element to run WEC-Sim for a monopile geometry. Execute WEC-Sim in either the v2020a or newer to run this case.  
+
```

Comment:
@akeeste can you please update this README and add more info about the example, then it's ready for a merge. Thanks!